src/components/VFooter/VFooter.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <footer class="flex flex-col gap-10 px-6 py-10 lg:px-10 xl:flex-row xl:gap-8">
+    <!-- Logo and links -->
+    <div
+      v-if="isContent"
+      class="flex flex-col gap-10 md:flex-row md:items-center md:justify-between xl:flex-grow"
+    >
+      <VBrand class="text-[18px]" />
```

Comment:
The Brand here should probably be a home page link.

Reply:
Right! Thanks, I'll add that.